app/(main)/league/[leagueId]/entry/all/page.test.tsx

```diff
@@ -171,4 +171,53 @@ describe('League entries page (Entry Component)', () => {
       expect(viewPastWeeksLink).toBeInTheDocument();
     });
   });
+
+  it('should display a button to add a new entry if there are less than 5 entries', async () => {
+    mockGetCurrentUserEntries.mockResolvedValueOnce([
+      {
+        id: '66311a210039f0532044',
+        week: 1,
+        selectedTeams: [],
+      },
+    ]);
+
+    render(<Entry params={{ leagueId: '66311a210039f0532044' }} />);
+
+    await waitFor(() => {
+      expect(screen.getByTestId('add-new-entry-button')).toBeInTheDocument();
+    });
+  });
+
+  it('should not display a button to add a new entry if there are more than 5 entries', async () => {
+    mockGetCurrentUserEntries.mockResolvedValueOnce([
+      {
+        id: '66311a210039f0532044',
+        week: 1,
+        selectedTeams: [],
+      },
+      {
+        id: '66311a210039f0532045',
+        week: 1,
+        selectedTeams: [],
+      },
+      {
+        id: '66311a210039f0532046',
+        week: 1,
+        selectedTeams: [],
+      },
+      {
+        id: '66311a210039f0532047',
+        week: 1,
+        selectedTeams: [],
+      },
```

Comment:
Correct me if I'm wrong, but if we're testing to make sure that the UI should not display a button to add a new entry if there are more than 5 entries, then shouldn't there be 5 entries here?

Reply:
good catch! oops!

Reply:
@choir27 good catch! this is why I shouldn't work and code review 😅

Reply:
@choir27 ✅ fixed.